ibm/data_source_ibm_iam_user_policy_test.go

```diff
@@ -18,7 +18,7 @@ func TestAccIBMIAMUserPolicyDataSource_Basic(t *testing.T) {
 			resource.TestStep{
 				Config: testAccCheckIBMIAMUserPolicyDataSourceConfig(name),
 				Check: resource.ComposeTestCheckFunc(
-					resource.TestCheckResourceAttr("data.ibm_iam_user_policy.testacc_ds_user_policy", "policies.#", "1"),
+					resource.TestCheckResourceAttr("data.ibm_iam_user_policy.testacc_ds_user_policy", "policies.#", "5"),
```

Comment:
Here also don't change the count...since you are running with our ID you may have different number of policies